release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1770,7 +1770,13 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, users with restricted access could not access their config map in a shared namespace to save their user settings on a cluster and load them in another browser or machine. As a result, user preferences such as pinned navigation items were only saved in the local browser storage and not shared between multiple browsers. This update resolves the issue: The web console Operator automatically creates RBAC rules so that each user can save these settings to a config map in a shared namespace and more easily switch between browsers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018234[BZ#2018234])
 
-* Before this update, creating connections between virtual machines (VMs) in the Topology view failed because it relied on a method that did not support custom resource definition (CRDs). This update resolves the issue by adding support for CRDs. Now you can create connections between VMs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020904[BZ#2020904])
+* Before this update, trying to create connections between virtual machines (VMs) in the *Topology* view failed with an "Error creating connection" message. This issue happened because this action relied on a method that did not support custom resource definition (CRDs). This update resolves the issue by adding support for CRDs. Now you can create connections between VMs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020904[BZ#2020904])
+
+* Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
+
+* Before this update, a layout issue caused the Web terminal to open outside the view and resize incorrectly. This update fixes the layout issue by updating the CSS. Now, the terminal opens inside the main view and resizes appropriately. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048598[BZ#2048598])
+
+* Before this update, a filtering issue displayed irrelevant and invalid tabs for workloads in the *Topology* view. This update fixes the issue by filtering those tabs before rendering the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053252[BZ#2053252])
```

Comment:
I don't think this issue was introduced and fixed in 4.10 itself and did not exist in prior release.
It must have been an oversight on my part.
Please remove this bug fix from the notes.